calendario/views.py

```diff
@@ -165,7 +165,7 @@ def get_appointments_list(servicio, user, **kwargs):
         kw['fin__lte'] = parse_datetime(end)
     if servicio is not None:
         kw['servicio__pk'] = servicio
-        kw['estado'] = 0
+        kw['estado__in'] = [0, 4]
```

Comment:
```suggestion
        kw['estado__in'] = [Turno.DISPONIBLE, Turno.CANCELADO_PACIENTE]
```
Asegurarse que `Turno` esté importado.
Ver además si se requiere `Turno.CANCELADO_ESTABLECIMIENTO`

Reply:
bien lo cambio por eso, me parece q agregar turno cancelado por el establecimiento estaria mal o hace falta agregar un tipo de cancelacion. Se supone que si lo cancela no se usa o un cancelado Medico?

Reply:
Todo lo que este cancelado se puede volver a usar. Ahora me haces pensar que tenemos que tener una forma de anular turnos, de sacarlos de agenda porque el medico se fue o porque por algún motivo no está mas abierto.
Por ahora si lo cancela, si se puede reasignar. Despues vemos si creamos otro estado o directamente permitimos eliminarlo
Fijate que podes aceptar el cambio usando el _commit suggestion_